src/app/contextBroker/contextBroker.cpp

```diff
@@ -191,7 +191,7 @@
 #include "cache/subCache.h"
 #include "cache/SubscriptionCache.h"
 
-
+using namespace orion;
```

Comment:
It depends... in some places you only need a punctual usage of the calss inside the namespace, so xxx::theClass is ok. In other places, you are continuously using the namespaced clase, and using xxx:: al the time make end in a noisy and less readable code.

I think it's a matter of common sense when to use one way or the other, so I wouldn't suggest not being too "normative" with this.